README.md

```diff
@@ -1,33 +1,34 @@
 [![Build Status](https://travis-ci.org/jquery/jquery-migrate.svg?branch=master)](https://travis-ci.org/jquery/jquery-migrate)
 [![Coverage Status](https://img.shields.io/coveralls/jquery/jquery-migrate.svg?style=flat)](https://coveralls.io/r/jquery/jquery-migrate?branch=master)
 
-# Migrate older jQuery code to jQuery 1.9+
+# Migrate older jQuery code to jQuery 3.0+
 
-This plugin can be used to detect and restore APIs or features that have been deprecated in jQuery and removed as of version 1.9.
+#### NOTE: If you have not yet upgraded to 1.12.x or 2.2.x, first upgrade to them and use [jQuery Migrate 1.x](https://github.com/jquery/jquery-migrate/tree/1.x-stable#readme) to resolve any compatibility issues before using jQuery Migrate 3.0 and upgrading to jQuery 3.0.
+
+This plugin can be used to detect and/or restore APIs and behaviors that have been deprecated and/or removed as of jQuery version 3.0.
```

Comment:
"detect and/or restore" doesn't read quite right... I think it should be "warn about and/or restore".